scripts/k8s-deployment-sim.sh

```diff
@@ -66,17 +66,16 @@ kubectl describe services $(kubectl get services|grep agent-plg|awk '{print $1}'
 printf "\n\n\n*** [3.4] clear installation\n"
 kubectl delete --all deployments && kubectl delete --all pods && kubectl delete --all services
 
-printf "\n\n\n*** [4] install lber in k8s\n"
-#helm install k8s/agent+lber --generate-name
-: 'printf "\n\n\n*** [4.1] verify installation\n"
+printf "\n\n\n*** [4] install oauth in k8s\n"
+helm install k8s/examples/oauth --set-file global.oauthConfig=k8s/examples/oauth/oauth.env --generate-name
```

Comment:
No pvc test available. Please verify in minikube

Reply:
pvc tests are added now.